heapster.go

```diff
@@ -35,8 +35,7 @@ import (
 )
 
 var (
-	argPollDuration    = flag.Duration("poll_duration", 10*time.Second, "The frequency at which heapster will poll for stats")
-	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats. Acceptable values are in the range [1 second, 'poll_duration')")
+	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats.")
```

Comment:
We currently use stats_resolution to down sample stats. Have you looked through the rest of the code to remove the downsampling, since we are choosing to poll more frequently?